spec/models/cartitem_spec.rb

```diff
@@ -5,9 +5,15 @@
   let(:r) { create_restaurant("Boston Beer Garden") }
   let(:cart) { FactoryGirl.create(:cart, user: user, restaurant: r) }
   let(:item) { FactoryGirl.create(:item, restaurantdetail: r.restaurantdetail) }
+  let(:longnameitem) { FactoryGirl.create(:item, :longname,
+                        restaurantdetail: r.restaurantdetail) }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 9, 63 should be on its own line.